CHANGELOG.md

```diff
@@ -627,7 +650,12 @@ Other minor additions
   <±-isStrictTotalOrder-≡      : IsStrictTotalOrder _≡_ _<_ → IsStrictTotalOrder _≡_ _<±_
   ```
 
-* In `Relation.Binary.HeterogeneousEquality` the relation `_≅_` has
+* Added new definitions in `Relation.Binary.Core`:
+  ```agda
+  Universal _∼_  = ∀ x y → x ∼ y
+  ```
```

Comment:
There seems to be a merge problem: you now have two entries for
`Relation.Binary.Core` in the CHANGELOG. This one is at the right
position for the alphabetical order but the other one lists the
right set of changes. Please merge them by hand.